lib/liquid/tags/include.rb

```diff
@@ -107,6 +109,15 @@ def read_template_from_file_system(context)
 
       file_system.read_template_file(context.evaluate(@template_name_expr))
     end
+
+    class ParseTreeVisitor < Liquid::ParseTreeVisitor
+      def children
+        [
+          @node.template_name_expr,
+          @node.variable_name_expr
+        ] + @node.attributes.values
```

Comment:
why only the `values`?

Reply:
the keys in the hash are the name of attributes to be assigned, not expressions